rust/core-lib/src/index_set.rs

```diff
@@ -18,23 +18,36 @@
 // Note: this data structure has nontrivial overlap with Subset in the rope
 // crate. Maybe we don't need both.
 
-use std::cmp::{max, min};
+use std::cmp::{max, min, Ordering};
 use xi_rope::{RopeDelta, Transformer};
 
 pub struct IndexSet {
     ranges: Vec<(usize, usize)>,
 }
 
 pub fn remove_n_at<T: Clone>(v: &mut Vec<T>, index: usize, n: usize) {
-    if n == 1 {
-        v.remove(index);
-    } else if n > 1 {
-        let new_len = v.len() - n;
-        for i in index..new_len {
-            v[i] = v[i + n].clone();
+    match n.cmp(&1) {
+        Ordering::Equal => {
+            v.remove(index);
         }
-        v.truncate(new_len);
+        Ordering::Greater => {
+            let new_len = v.len() - n;
+            for i in index..new_len {
+                v[i] = v[i + n].clone();
+            }
+            v.truncate(new_len);
+        }
+        Ordering::Less => {}
```

Comment:
total nit: it's idiomatic to write this as,

```rust
Ordering::Less => (),
```

which implies that you are explicitly returning the unit (`()`) and not returning it implicitly with an empty block.

Reply:
:ok_hand: